docs/ios/bazel.md

```diff
@@ -0,0 +1,230 @@
+# iOS Build Instructions - Bazel
+
+The following instructions describe how to add Product Science instrumentation to a [Bazel](https://bazel.build/) project's iOS application.
+
+!!! example
+
+    Several of this document's steps include an "Example" section that describes how to apply the instructions to a demo Bazel application named SimpleApp. If you'd like to follow along, just clone our [demoapps repository](https://github.com/product-science/demoapps):
+
+    ```shell
+    % cd ~
+    % git clone https://github.com/product-science/demoapps
+    ```
+
+    Our examples will assume that you've cloned the repository to your home directory and kept the repository's default directory name (`demoapps`). If you decide to change either of these details, then modify the specified paths to fit your setup.
+
+## 1. Copy `.productscience.yaml` {TODO-02} {TODO-08} to your home directory
+
+Product Science will provide you with a `.productscience.yaml` {TODO-02} file that contains your credentials and configuration details. 
+
+!!! info 
+    
+    If you haven't received this file or need to request a new copy, please reach out to your designated integration support contact via email or Slack. If this contact is unavailable, send an email to support@productscience.ai for assistance.
+
+Once you've received your `.productscience.yaml` {TODO-02} file, copy the file to your home directory {TODO-03}.
+
+??? example
```

Comment:
Collapsed:
<img width="937" alt="image" src="https://github.com/product-science/docs/assets/967049/ccd7681b-ed02-4fbf-9ce6-8359085e1e09">

Expanded:
<img width="943" alt="image" src="https://github.com/product-science/docs/assets/967049/eddca6c7-b71f-45fa-8d84-22063a4c32a6">

The other `??? example` sections have the same appearance.